gridstatus/isone.py

```diff
@@ -227,6 +229,92 @@ def get_load_forecast(self, date, end=None, verbose=False):
 
         return df
 
+    @support_date_range(frequency="DAY_START")
+    def get_solar_forecast(self, date, end=None, verbose=False):
+        """Return solar forecast published on a specific date
+        https://www.iso-ne.com/isoexpress/web/reports/operations/-/tree/seven-day-solar-power-forecast
+        """
+        return self._get_solar_or_wind_forecast(
+            date,
+            end,
+            resource_type="Solar",
+            verbose=verbose,
+        )
+
+    @support_date_range(frequency="DAY_START")
+    def get_wind_forecast(self, date, end=None, verbose=False):
+        """Return wind forecast published on a specific date
+        https://www.iso-ne.com/isoexpress/web/reports/operations/-/tree/seven-day-wind-power-forecast
+        """
+        return self._get_solar_or_wind_forecast(
+            date,
+            end,
+            resource_type="Wind",
+            verbose=verbose,
+        )
+
+    def _get_solar_or_wind_forecast(self, date, end, resource_type, verbose=False):
```

Comment:
Example output for historical date range:


```python
from gridstatus.isone import ISONE

ISONE().get_solar_forecast("2024-02-10", "2024-02-12")
```

| Interval Start            | Interval End              | Publish Time              | Solar Forecast | 
|---------------------------|---------------------------|---------------------------|----------------| 
| 2024-02-18 04:00:00-05:00 | 2024-02-18 05:00:00-05:00 | 2024-02-11 20:29:34-05:00 | 156            | 
| 2024-02-18 05:00:00-05:00 | 2024-02-18 06:00:00-05:00 | 2024-02-11 20:29:34-05:00 | 152            | 
| 2024-02-18 06:00:00-05:00 | 2024-02-18 07:00:00-05:00 | 2024-02-11 20:29:34-05:00 | 149            | 
| 2024-02-18 07:00:00-05:00 | 2024-02-18 08:00:00-05:00 | 2024-02-11 20:29:34-05:00 | 153            | 
| 2024-02-18 08:00:00-05:00 | 2024-02-18 09:00:00-05:00 | 2024-02-11 20:29:34-05:00 | 152            | 
| 2024-02-18 09:00:00-05:00 | 2024-02-18 10:00:00-05:00 | 2024-02-11 20:29:34-05:00 | 139            |